dashboard/settings.py

```diff
@@ -31,6 +31,7 @@
 
 # Google Analytics ID
 GA_ID = config('GA_ID', default='')
+SITE_ID = 1
```

Comment:
What's this value? I don't see it used.

Reply:
The documentation says it is needed. since we don't run multiple apps in Django the default seems to be 1. https://simpleisbetterthancomplex.com/tutorial/2016/10/04/how-to-use-django-flatpages-app.html

Reply:
Cool, could you put a comment here that SITE_ID is needed for flatpages so we remember?

Reply:
Done